### DISCUSSION TITLE
Add permissions for each commerce widget

### DISCUSSION THREAD

**Discussion by Anubarak** [craftcms/commerce#2406] [Ideas]
**What are you trying to do?**
I created custom permissions for each Commerce widget if a user is allowed to see that or not. 
The disadvantage of that is: I always need to check all your widgets during each update to add the permission.
Ideally you would create permissions for each of them if a user is able to see it or not.

Use case: The Shop owner does not want to expose the entire data to all employees. Of course they could still calculate all orders by themself but it's more difficult to do that.

**Comment by lukeholder** [comment]
Thanks for the feedback. Keeping this open as a feature request.
